### ISSUE TITLE
Could not find a valid gem 'dpl-cloudfoundry' (= 1.9.1.travis.2639.5) in any repository

### ISSUE THREAD

**Issue by dzeyms** [travis-ci/travis-ci#9280]
I recently got this error in travis while deploying application in cloud foundry. Any ideas?

```
Installing deploy dependencies

ERROR:  Could not find a valid gem 'dpl-cloudfoundry' (= 1.9.1.travis.2639.5) in any repository

ERROR:  Possible alternatives: cloudfoundry, bosh-cloudfoundry, cloudfoundry-env

/home/travis/.rvm/rubies/ruby-2.2.7/lib/ruby/site_ruby/2.2.0/rubygems/core_ext/kernel_require.rb:59:in `require': cannot load such file -- dpl/provider/cloud_foundry (LoadError)

	from /home/travis/.rvm/rubies/ruby-2.2.7/lib/ruby/site_ruby/2.2.0/rubygems/core_ext/kernel_require.rb:59:in `require'

	from /home/travis/.rvm/gems/ruby-2.2.7/gems/dpl-1.9.1.travis.2639.5/lib/dpl/provider.rb:93:in `rescue in block in new'

	from /home/travis/.rvm/gems/ruby-2.2.7/gems/dpl-1.9.1.travis.2639.5/lib/dpl/provider.rb:67:in `block in new'

	from /home/travis/.rvm/gems/ruby-2.2.7/gems/dpl-1.9.1.travis.2639.5/lib/dpl/cli.rb:41:in `fold'

	from /home/travis/.rvm/gems/ruby-2.2.7/gems/dpl-1.9.1.travis.2639.5/lib/dpl/provider.rb:66:in `new'

	from /home/travis/.rvm/gems/ruby-2.2.7/gems/dpl-1.9.1.travis.2639.5/lib/dpl/cli.rb:31:in `run'

	from /home/travis/.rvm/gems/ruby-2.2.7/gems/dpl-1.9.1.travis.2639.5/lib/dpl/cli.rb:7:in `run'

	from /home/travis/.rvm/gems/ruby-2.2.7/gems/dpl-1.9.1.travis.2639.5/bin/dpl:5:in `<top (required)>'

	from /home/travis/.rvm/gems/ruby-2.2.7/bin/dpl:23:in `load'

	from /home/travis/.rvm/gems/ruby-2.2.7/bin/dpl:23:in `<main>'

failed to deploy
```

**Comment by BanzaiMan** [comment]
When reporting a problem, please include a build log URL that shows the problem you are describing. If it’s for a private repository and (or for any reason) you’d rather not discuss it in public, please close this issue and send email to support@travis-ci.com.

Thank you!

**Comment by wyemun** [comment]
Similar error I encounter as well, but instead of `dpl-cloudfoundry`, mine was `dpl-elastic-beanstalk`

**Comment by dzeyms** [comment]
@wyemun yeah it was working some time last week. For some reason, it's throwing this error. Any luck on your part?

**Comment by dzeyms** [comment]
Closing this issue and will send an email instead.

**Comment by BanzaiMan** [comment]
This should now be fixed by `1.9.1.travis.2660.5` and later. In general, though, `edge` means unstable. Things can break from time to time.
